crates/iota-types/src/lib.rs

```diff
@@ -95,7 +95,6 @@ pub mod versioned;
 pub mod zk_login_authenticator;
 pub mod zk_login_util;
 
-#[cfg(any(test, feature = "test-utils"))]
```

Comment:
Shouldn't it be the following?
```diff
-#[cfg(any(test, feature = "test-utils"))]
+#[cfg(test)]
```
There are other occurrences of that in your PR. Do we need to change those?

Reply:
So this reason it is like this is because these are publicly accessible from other crates. The feature previously was used because of this, but it is also valid internally to the crate for testing so both were used.

Reply:
Ah alright, makes sense 👍